api.swagger.yaml

```diff
@@ -6140,6 +6142,18 @@ components:
         - title
       type: object
     user-schema:
+      allOf:
+        - $ref: '#/components/schemas/user-schema-common'
+        - properties:
+            authname:
+              description: 'The authname (SSO unique ID) of the user.'
+              type: object
+              properties:
+                saml:
+                  description: 'The SAML containing the authname.'
+                  type: string
+                  example: '<saml:Assertion ...> ... </saml:Assertion>'
```

Comment:
This is just a simple string. Let's make the example "N2IwNTFmYjYtOGQ2MS00Yjg0LTlhMTgtMzRkY2M5OTFhNDI2"

This is a UUID ran through base64 conversion, there is no validation needed here as we can just allow string values, but this is an example of a good authname to prevent user login issues.